src/css/styles.css

```diff
@@ -2,6 +2,126 @@
 @import './global.css';
 
 body {
-  font-family: 'Nunito', sans-serif; /* Using Nunito font family*/
-  background-color: var(--primary-color); /*using a css variable*/
+  background-color: var(--light-blue-color);
 }
+
+/* MEALS STYLES */
+figure {
+  margin: 0;
+}
+
+.meals {
+  max-width: 774px;
+  margin: auto;
+}
+
+.title {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 24px;
+  color: var(--dark-gray-color);
+  margin-left: 10px;
+}
+
+.meals-content {
+  display: grid;
+  grid-template: auto / repeat(3, 33.3%);
+  align-items: center;
+}
+
+.meals-card {
+  background-color: var(--white-color);
+  margin: 10px;
+  padding: 10px;
+  border-radius: 5px;
+  box-shadow: 0 5px 5px var(--light-gray-color);
+}
+
+.meals-card p {
+  font-family: 'Permanent Marker', cursive;
+  font-size: 14px;
+  color: var(--black-color);
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: center;
+}
+
+.meals-card img {
+  width: 100%;
```

Comment:
Maybe you can add the property `object-fit: cover` just to make sure that every image, no matter the size,  is going to cover all of the container space and not is going to distort.

Reply:
Perfect! Thank you. Changes already done.